Teapot/MockTeapot.swift

```diff
@@ -22,38 +20,56 @@ open class MockTeapot: Teapot {
         case invalidMockFile(String)
     }
 
-    /// The status code the URL request should return
+    private let currentBundle: Bundle
+    private let resource: String
     private let statusCode: StatusCode
 
+    private var overrideEndpointDictionary = [String: String]()
+
     /// Initialiser.
     ///
     /// - Parameters:
-    ///   - bundle: the bundle of your test target. When you add a json file with the name of the endpoint to your test target it will return this data.
+    ///   - bundle: the bundle of your test target, where it will search for the mock file
+    ///   - mockFileName: the name of the mock file containing the json that will be returned
     ///   - statusCode: the status code for the response to return errors. Default is 200 "ok" 👌
-    public init(bundle: Bundle, statusCode: StatusCode = .ok) {
+    public init(bundle: Bundle, mockFileName: String, statusCode: StatusCode = .ok) {
         self.currentBundle = bundle
+        self.resource = mockFileName
```

Comment:
`resource` is a bit vague. If you're setting it always to the mock filename, why not name them the same?